app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
+    if command.split()[0] != "mv":
+        return
+    cmd, file_from, file_to = command.split()
+    if "/" in file_to:
```

Comment:
Is this is Windows - we will have `\` in the path. Please, try not to use this. Think about different variations. What about os.path.dirname

Reply:
I have to add "\" in condition to check if it's the path to directory (on windows) But how to implement if? Because this symbol is already reserved by python as a string symbol

Reply:
ok,googled it.
I hope everything is correct